### PR TITLE
Add NetApp SolidFire check connections module

### PR DESCRIPTION
**Pull request by timuster**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

NetApp SolidFire check connections module

##### ANSIBLE VERSION

```
ansible 2.3.0
  config file = 
  configured module search path = Default w/o overrides
```

##### SUMMARY

Adding sf_check_connections.py, a module to check NetApp SolidFire connections. The design was based on the existing NetApp E-Series modules.

```

```

### CODE REVIEW

**Comment by hulquest** [comment]
All of the general comments made in #20732 have been applied here.  
The module is sound from a SF API point of view. 
shipit

File: lib/ansible/modules/storage/netapp/sf_check_connections.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+EXAMPLES = """
+   - name: Check connections to MVIP and SVIP
+     sf_check_connections:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+"""
+
+RETURN = """
+msg:
```
Comment (resmo) [inline lib/ansible/modules/storage/netapp/sf_check_connections.py]: It is not in the correct format and only documents common returns, which must not be documented. Please remove this return content, but keep the variable, e.g.
~~~ yaml
RETURN = """
"""
~~~

**Comment by resmo** [comment]
Minors with docs and missing metadata, otherwise lgtm

shipit

**Comment by resmo** [comment]
shipit

**Comment by hulquest** [comment]
shipit

**Comment by resmo** [comment]
shipit

**Comment by lmprice** [comment]
I'm definitely not a SF guy, but it looks sound to me.
shipit

**Pull request merged**